modules/install/pages/rhel-suse-install-intro.adoc

```diff
@@ -119,87 +141,113 @@ Available releases are listed with their full `version-build` number:
 +
 [subs=+quotes]
 ----
-couchbase-server-community.x86_64   *6.0.0-1693*
+couchbase-server-community.x86_64 <version_number>-<build_number>
 ----
 +
+The following is an example of the listing:
++
+[subs=+quotes]
+----
+couchbase-server-community.x86_64 *8.0.0-3777*
+----
+
 . Specify a release to install it.
 +
 [source,console,subs=+quotes]
 ----
-sudo yum install couchbase-server-community-[.var]_version-build_
+sudo yum install couchbase-server-community-<version_number>-<build_number>
 ----
 +
-Using the example listing from the previous step, the resulting installation command would be:
+The following is an example of the installation command:
 +
 [subs=+quotes]
 ----
-sudo yum install couchbase-server-community-*6.0.0-1693*
+sudo yum install couchbase-server-community-*8.0.0-3777*
 ----
 +
-You'll be prompted to start the download of Couchbase Server (plus any dependencies), as well as import several GPG keys.
+You'll be prompted to start the download of Couchbase Server and its dependencies, and import several GPG keys.
 For each of these prompts, type `y` to accept and continue.
 --
 ====
 +
-Once installation is complete, Couchbase Server will start automatically (and will continue to start automatically at run levels 2, 3, 4, and 5, and explicitly shut down at run levels 0, 1, and 6).
-You can use the `systemctl` command (`service` on older operating systems) to start and stop the Couchbase Server service, as well as check the current status.
-Refer to xref:startup-shutdown.adoc[Couchbase Server Startup and Shutdown] for more information.
-+
-. Open a web browser and access the Couchbase Web Console to xref:testing.adoc[verify] that the installation was successful and that the node is available.
+After the installation is complete, Couchbase Server starts automatically.
+Couchbase Server continues to start automatically at run levels 2, 3, 4, and 5, and explicitly shuts down at run levels 0, 1, and 6.
+You can use the `systemctl` command, a `service` on older operating systems, to start and stop the Couchbase Server service, and to verify the current status.
+For more information, see xref:startup-shutdown.adoc[Couchbase Server Startup and Shutdown].
+
+. Open the Couchbase Web Console in a browser to xref:testing.adoc[verify] that the installation was successful and that the node is available.
 
 === Install Using RPM Package
 
```

Comment:
I guess AL2023 doesn't support yum.  Is it why you list install commands separately for each platform?  This section could be simplified by using 'dnf' instead of yum & rpm.  'dnf' is supported on Red Hat Enterprise or Oracle Linux or Amazon Linux 2023.  Need to verify on all platforms though.
"sudo dnf install ./package.rpm"